### PR TITLE
docs: notification added

### PR DESCRIPTION
**Pull request by rahulyadav139**

### CODE REVIEW

**Review by ravikumar1002 — APPROVED** [review]
looks good to merge

File: src/components/header/Navigation.css [hunk 1]
```diff
@@ -1,33 +1,44 @@
...
   padding-bottom: 0.2rem;
   margin-top: 0.5rem;
-}
+} */
+
+/* .notification-container {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+} */
```
Comment (HarshnaNHaswani) [inline src/components/header/Navigation.css:41]: remove commented code before committing

File: src/components/header/Navigation.css [hunk 2]
```diff
@@ -1,33 +1,44 @@
...
-.nav-bar .profile-item:hover .profile-item-dropdown {
-  display: block;
-}
 
-.profile-item-dropdown {
+.notification-container {
   position: absolute;
   top: 3rem;
-  right: 0;
+  right: -100px;
```
Comment (HarshnaNHaswani) [inline src/components/header/Navigation.css:9]: can use rem here too

File: src/components/header/Navigation.js [hunk 3]
```diff
@@ -1,9 +1,15 @@
 import './Navigation.css';
+import { Notification } from '../notification/Notification';
 
 const Navigation = props => {
   return (
     <nav>
       <ul className="list-items">
+        <li className="list-item">
+          <button className="btn icon primary medium">
+            <i class="fas fa-home"></i>
```
Comment (HarshnaNHaswani) [inline src/components/header/Navigation.js:10]: remove props if its not needed

**Review by HarshnaNHaswani — APPROVED** [review]
Looks good to merge
Clean code
Easily understandable

**Pull request merged**